Validation/nightly/extracted_04.fcl

```diff
@@ -1,6 +1,6 @@
 #include "Production/JobConfig/reco/Extracted.fcl"
 services.DbService.purpose: MDC2020_perfect
-services.DbService.version: v1_0
+services.DbService.version: v1_3
```

Comment:
Why did this jump to v1_3 instead of v1_2?  does v1_3 exist?  Should this come from database.fcl?

Reply:
This is MDC2020_perfect instead of best since it runs on perfect digis right now

Reply:
Generally we reserve perfect for algorithm debugging.  Is there a reason besides history extracted validation isn't using best?

Reply:
It's only historical, and should be updated soon with a newer list of input files

Reply:
I just tested that changing extracted_0*.fcl to use MDC2020_best, v1_2 runs without errors and produces reasonable results.  Is there an inconsistency I'm missing?
If we could consolidate all the nightly scripts to use database.fcl now that would be better, even if the plan is to replace them soon.

Reply:
Ok updated for now, will remember that I still need to check on updating these inputs